README.md

```diff
@@ -1,2 +1,70 @@
-# Bookstore-React
-An online bookstore built with react.
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
```

Comment:
- [ ] Please explain to the users how can they clone and run your project in details

Reply:
Alright, forgot to change the README.md file. Thanks for pointing out.

Reply:
- [ ] Please be descriptive about how users can clone and run your project. Make it step by step.